### PR TITLE
Remove type level trimming from BlazorWebAssembly projects

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by eerhardt** [comment]
I think we should wait for this change until the underlying linker changes are fully flown though the system. That way we don't regress size by not trimming these assemblies at all.

@sbomer - do you know when the linker changes for IsTrimmable will be available in the main branch?

**Comment by sbomer** [comment]
I'm close to ready with the SDK change and will have a PR out soon. Do you know if the SDK is still taking changes from master into the P2 branch? (If they're not I can go ahead and check in the linker change so that it starts flowing).

**Comment by pranavkm** [comment]
> Do you know if the SDK is still taking changes from master into the P2 branch? (If they're not I can go ahead and check in the linker change so that it starts flowing).

They branched. Master is targeting preview3.

File: src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.Current.targets [hunk 1]
```diff
@@ -421,7 +423,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     </Copy>
   </Target>
 
-  <Target Name="_BlazorWasmPrepareForLink" BeforeTargets="PrepareForILLink">
+  <!--
+    This target special trimming for Microsoft.Extensions.* and Microsoft.AspNetCore.* assemblies.
```
Comment (pranavkm) [inline src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.Current.targets:427]:
```suggestion
    This target configures special trimming for Microsoft.Extensions.* and Microsoft.AspNetCore.* assemblies.
```

**Comment by pranavkm** [comment]
@sbomer could you ping me on the other PR so we can get this in right after?

**Comment by pranavkm** [comment]
@sbomer good to go?

**Comment by sbomer** [comment]
@pranavkm yep, go for it!

**Pull request merged**